otter/convergence/gathering.py

```diff
@@ -196,25 +197,31 @@ def update_drained_at(node):
     yield do_return(list(filter(bool, nodes)))
 
 
+_DRAINING_CREATED_RE = (
```

Comment:
To be clear: it's just sad that this is the only way to do this.  I wonder if it would be appropriate to open a ticket on CLBs to get this information in a more structured way, sans regexes (if you haven't already).